### PR TITLE
Feature/normalization of beattable 253

### PR DESCRIPTION
**Pull request by valentinbujdoso**

### CODE REVIEW

**Review by magwas — CHANGES_REQUESTED** [review]
sonar is right on all two counts

**Comment by valentinbujdoso** [comment]
Thanks fixed.

**Review by magwas — APPROVED** [review]

**Comment by magwas** [comment]
testcasecount=6

File: src/main/java/org/rulez/demokracia/pdengine/Normalizable.java [hunk 1]
```diff
@@ -9,6 +13,27 @@ default void normalize() {
...
+					int key1Los = beatInformation(key1, key2, Direction.DIRECTION_BACKWARD);
+					int key2Los = beatInformation(key2, key1, Direction.DIRECTION_BACKWARD);
+
+					if (key1Los == key2Los) {
+						setElement(key2, key1, new Pair(0, 0));
+						setElement(key1, key2, new Pair(0, 0));
+					}
+				}
+			}
+		}
```
Comment (szirbucz) [inline src/main/java/org/rulez/demokracia/pdengine/Normalizable.java:36]: I would use compareBeats instead of re-implementing its functionality.

**Review by szirbucz — CHANGES_REQUESTED** [review]

File: src/main/java/org/rulez/demokracia/pdengine/Normalizable.java (hunk 1 above)
Comment (szirbucz) [reply src/main/java/org/rulez/demokracia/pdengine/Normalizable.java:36]: Too late :)

Comment (magwas) [reply src/main/java/org/rulez/demokracia/pdengine/Normalizable.java:36]: Valentin grabbed it before I could comment here

**Pull request merged**